public-docs/installation-reaction-platform.md

```diff
@@ -0,0 +1,104 @@
+---
+id: installation-reaction-platform
+title: Install with Reaction Platform
+---
+
+Follow these instructions to run the entire Reaction Platform. 
+
+The Platform will install and run the the entire suite of Reaction services in the these directories:
+
+| Directory                                                                                  | Services                        |
+| ------------------------------------------------------------------------------------------ | ------------------------------- |
+| [`reaction`](https://github.com/reactioncommerce/reaction)                                 | GraphQL API, Classic UI, Mongo  |
+| [`reaction-hydra`](https://github.com/reactioncommerce/reaction-hydra)                     | Authentication server, Postgres |
+| [`reaction-next-starterkit`](https://github.com/reactioncommerce/reaction-next-starterkit) | Next.js storefront application  |
```

Comment:
@ticean - Does the way I broke down the apps by directory, and then listed each service, look good to you?

And as for naming, how is `GraphQL API, Classic UI, Mongo`? Or would you prefer: `Reaction API, Mongo, Classic UI` or something else?